### PR TITLE
Periodic process to clear Packet leaks

### PR DESCRIPTION
**Pull request by akiselev1**
Baremetalds packet garbage detection test PR

### CODE REVIEW

**Comment by akiselev1** [comment]
/retitle  [WIP] Periodic process to clear Packet leaks

**Comment by akiselev1** [comment]
Reopening this PR after a force-push

**Comment by akiselev1** [comment]
Reopening this PR after a force-push

File: ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    exit 1
+fi
+
+#Packet API call to get list of servers in project
+servers="$(curl -X GET --header 'Accept: application/json' --header "X-Auth-Token: ${PACKET_AUTH_TOKEN}"\
+ "https://api.packet.net/projects/${PACKET_PROJECT_ID}/devices?exclude=root_password,ssh_keys,created_by,project,project_lite\
+,ip_addresses,plan,meta,operating_system,facility,network_ports&per_page=1000")"
+
+#Assuming all servers created more than 4 hours = 14400 sec ago are leaks
+leaks="$(echo "$servers" | jq -r '.devices[]|select((now-(.created_at|fromdate))>14400)')"
```
Comment (stbenjam) [inline ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh:29]: I know @andfasano added some tags/labels to our hosts -- is there a way to differentiate jobs from that come of a CI job vs. one that comes from @cluster-bot? The latter last 12 hours I think.

We can solve it later I suppose, 4 hours is more than enough for now

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh:29]: (We don't have cluster-bot setup yet for e2e-metal-ipi, but hopefully soon)

Comment (akiselev1) [reply ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh:29]: Tags were temporary removed after moving to ansible and new packet project. That's my next task. Need to add labels handling to packet ansible module, then re-introduce them here.

**Pull request merged**